device/celestica/x86_64-cel_silverstone-r0/Silverstone-128x100/minigraph.xml

```diff
@@ -0,0 +1,1333 @@
+<DeviceMiniGraph xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="Microsoft.Search.Autopilot.Evolution">
```

Comment:
No minigraph.xml files are needed as mentioned by Guohan too.

Reply:
We will remove Minigraph

Reply:
fixed in bf1a418.